tests/DCPS/XTypes/run_test.pl

```diff
@@ -0,0 +1,66 @@
+eval '(exit $?0)' && eval 'exec perl -S $0 ${1+"$@"}'
+     & eval 'exec perl -S $0 $argv:q'
+     if 0;
+
+use lib "$ENV{ACE_ROOT}/bin";
+use lib "$ENV{DDS_ROOT}/bin";
+use PerlDDS::Run_Test;
+use Getopt::Long;
+use strict;
+
+my $test = new PerlDDS::TestFramework();
+$test->enable_console_logging();
+
+my $verbose = 0;
+my $test_name = "";
+GetOptions(
+  "verbose" => \$verbose,
+  "test|f=s"   => \$test_name,
```

Comment:
```suggestion
  "test|f=s" => \$test_name,
```

Also why `-f`? Did you mean `-t`?